src/Symfony/Component/Routing/Loader/DependencyInjection/ServiceRouterLoader.php

```diff
@@ -11,28 +11,13 @@
 
 namespace Symfony\Component\Routing\Loader\DependencyInjection;
 
-use Psr\Container\ContainerInterface;
-use Symfony\Component\Routing\Loader\ObjectRouteLoader;
+use Symfony\Component\Routing\Loader\ContainerLoader;
+
+@trigger_error(sprintf('The "%s" class is deprecated since Symfony 4.4 and will be removed in 5.0, use "%s" instead.', ServiceRouterLoader::class, ContainerLoader::class), E_USER_DEPRECATED);
 
 /**
- * A route loader that executes a service to load the routes.
- *
- * @author Ryan Weaver <ryan@knpuniversity.com>
+ * @deprecated since Symfony 4.4 and will be removed in 5.0, use Symfony\Component\Routing\Loader\ContainerLoader instead.
```

Comment:
all "and will be removed" should be removed from the PR, we try to get rid of them